sections/ads.txt

```diff
@@ -222,8 +222,10 @@ borsonline.hu##A[href="/jatek"]
 borsonline.hu##A[href="/nyarimagazin"]
 borsonline.hu##a[target="_blank"][class="videa-wrapper"]
 borsonline.hu##iframe[src*="koponyeg.hu"]
```

Comment:
```suggestion
borsonline.hu##iframe[src*="koponyeg.hu"]
bpiautosok.hu##[class="br"]
```

Your proposed rules work fine with uBO and AG. However, they do not work with Brave's adblocker, so I suggest adding `bpiautosok.hu##[class="br"]`. [Adblock-rust](https://github.com/brave/adblock-rust/) is not listed as a supported blocker in our readme though, so feel free to ignore my suggestion. Plus, it might cause breakage in the future, but for now it works fine.

Reply:
@ehrhardt06 Rules using `:first-child` not working in Brave adblock?

Reply:
@scripthunter7 I checked again, it works, but only after refreshing the page. So after you visit the site for the first time, the white bar does not disappear, you have to refresh the page or click somewhere, then the white bar disappears. This is what fooled me, uBO (or Firefox) does not seem to do this...

Reply:
@ehrhardt06 Sounds intresting, I'll try it with Brave in the morning. If you have any ideas until then, please write a comment here

Reply:
It's indeed interesting. So after you visit the site for the first time, brave doesn't block, but after you refresh the page or visit an article, it disappears thanks to your filters, and the white bar never appears again. But, if I clear all browsing data (cache, cookies etc), the white bar appears again, then after a refresh, it disappears again. What is even more strange is that sometimes even after clearing the browsing data, brave successfully blocks it for the first site visit. I tried aggressive and standard blocking in shields, but the result is the same. I'm not sure if it's a bug or not, maybe I'm missing something. Nevertheless, it's not a big issue, since the rules work for most of the time, and it's only an issue with this browser.

Reply:
@ehrhardt06 tried with hufilter + custom rules and custom rules only, but can't reproduce the issue

Reply:
Not sure whats the problem on my end then @scripthunter7, but it's not important.

These can be merged then:

```adblock-filter-list
bpiautosok.hu##body > center:first-child
bpiautosok.hu##body > center:first-child + br
bpiautosok.hu##font + center
```